converter/internal/prometheusconvert/remote_write.go

```diff
@@ -25,12 +25,12 @@ func toRemotewriteArguments(promConfig *prom_config.Config) *remotewrite.Argumen
 
 	return &remotewrite.Arguments{
 		ExternalLabels: externalLabels,
-		Endpoints:      getEndpointOptions(promConfig.RemoteWriteConfigs),
+		Endpoints:      GetEndpointOptions(promConfig.RemoteWriteConfigs),
 		WALOptions:     remotewrite.DefaultWALOptions,
 	}
 }
 
-func getEndpointOptions(remoteWriteConfigs []*prom_config.RemoteWriteConfig) []*remotewrite.EndpointOptions {
```

Comment:
how do we feel about this vs copy/pasting a bunch of this code into the new package?

Reply:
Doesn't matter, it's internal :)